root/static/scripts/release-editor/seeding.js

```diff
@@ -75,7 +75,11 @@ releaseEditor.seedRelease = function (release, data) {
     }
 
     if (data.barcode) {
-        release.barcode.value(data.barcode);
```

Comment:
I kinda think we should handle this on the server by doing something like `$result->{barcode} = '' if $result->{barcode} eq 'none';` below https://github.com/metabrainz/musicbrainz-server/blob/3702485a813fa82ce894a555c7b6a9b9a40447e8/lib/MusicBrainz/Server/Controller/ReleaseEditor.pm#L148-L148
since that's where the other seeded data processing is done at present. Though that requires changing the `data.barcode` check in the JS above to `!= null`.

Reply:
(Decided in IRC to do something different, since it wasn't as simple as that.)